### ISSUE TITLE
Abstract Method Error

### ISSUE THREAD

**Issue by kshivang** [rever-ai/Goonj#5]
```
java.lang.AbstractMethodError: abstract method "java.lang.String com.google.android.exoplayer2.ui.PlayerNotificationManager$MediaDescriptionAdapter.getCurrentSubText(com.google.android.exoplayer2.Player)"
        at com.google.android.exoplayer2.ui.PlayerNotificationManager.createNotification(PlayerNotificationManager.java:1038)
        at com.google.android.exoplayer2.ui.PlayerNotificationManager.startOrUpdateNotification(PlayerNotificationManager.java:918)
        at com.google.android.exoplayer2.ui.PlayerNotificationManager.startOrUpdateNotification(PlayerNotificationManager.java:910)
        at com.google.android.exoplayer2.ui.PlayerNotificationManager.access$1000(PlayerNotificationManager.java:108)
        at com.google.android.exoplayer2.ui.PlayerNotificationManager$PlayerListener.onPlayerStateChanged(PlayerNotificationManager.java:1282)
        at com.google.android.exoplayer2.ExoPlayerImpl$PlaybackInfoUpdate.lambda$run$4$ExoPlayerImpl$PlaybackInfoUpdate(ExoPlayerImpl.java:812)
        at com.google.android.exoplayer2.-$$Lambda$ExoPlayerImpl$PlaybackInfoUpdate$sJrY7lA_vUJy5MdfV-ndTSxVTXI.invokeListener(Unknown Source:2)
        at com.google.android.exoplayer2.BasePlayer$ListenerHolder.invoke(BasePlayer.java:165)
        at com.google.android.exoplayer2.ExoPlayerImpl.invokeAll(ExoPlayerImpl.java:823)
        at com.google.android.exoplayer2.ExoPlayerImpl.access$000(ExoPlayerImpl.java:43)
        at com.google.android.exoplayer2.ExoPlayerImpl$PlaybackInfoUpdate.run(ExoPlayerImpl.java:810)
        at com.google.android.exoplayer2.ExoPlayerImpl.notifyListeners(ExoPlayerImpl.java:726)
        at com.google.android.exoplayer2.ExoPlayerImpl.updatePlaybackInfo(ExoPlayerImpl.java:701)
        at com.google.android.exoplayer2.ExoPlayerImpl.prepare(ExoPlayerImpl.java:232)
        at com.google.android.exoplayer2.SimpleExoPlayer.prepare(SimpleExoPlayer.java:913)
        at com.google.android.exoplayer2.SimpleExoPlayer.prepare(SimpleExoPlayer.java:898)
        at ai.rever.goonj.player.LocalAudioPlayer.enqueue(LocalAudioPlayer.kt:236)
        at ai.rever.goonj.manager.GoonjPlayerManager.addTrack(GoonjPlayerManager.kt:79)
        at ai.rever.goonj.manager.GoonjPlayerManager.addTrack$default(GoonjPlayerManager.kt:75)
        at ai.rever.goonj.Goonj$addTrack$1.invoke(Goonj.kt:149)
        at ai.rever.goonj.Goonj$addTrack$1.invoke(Goonj.kt:32)
        at ai.rever.goonj.Goonj.run(Goonj.kt:62)
        at ai.rever.goonj.Goonj.addTrack(Goonj.kt:146)
        at ai.rever.goonj.interfaces.GoonjPlayer$DefaultImpls.addTrack(GoonjPlayer.kt:26)
        at ly.blissful.bliss.ui.commons.player.audio.v2.GoonjPlayerActivity.addTrack(GoonjPlayerActivity.kt:29)
        at ai.rever.goonj.interfaces.GoonjPlayer$DefaultImpls.addTrack$default(GoonjPlayer.kt:26)
        at ly.blissful.bliss.ui.commons.player.audio.v2.GoonjPlayerActivity.playNewTrack(GoonjPlayerActivity.kt:212)
        at ly.blissful.bliss.ui.commons.player.audio.v2.GoonjPlayerActivity$requestStart$2.accept(GoonjPlayerActivity.kt:197)
        at ly.blissful.bliss.ui.commons.player.audio.v2.GoonjPlayerActivity$requestStart$2.accept(GoonjPlayerActivity.kt:29)
        at io.reactivex.internal.operators.maybe.MaybeCallbackObserver.onSuccess(MaybeCallbackObserver.java:71)
        at io.reactivex.internal.operators.maybe.MaybeMap$MapMaybeObserver.onSuccess(MaybeMap.java:89)
        at io.reactivex.internal.operators.maybe.MaybeOnErrorNext$OnErrorNextMaybeObserver.onSuccess(MaybeOnErrorNext.java:88)
        at io.reactivex.internal.operators.maybe.MaybeCreate$Emitter.onSuccess(MaybeCreate.java:73)
        at ly.blissful.bliss.api.DocumentGetterKt$getDocument$1$1.onSuccess(DocumentGetter.kt:48)
        at ly.blissful.bliss.api.DocumentGetterKt$getDocument$1$1.onSuccess(Unknown Source:2)
        at com.google.android.gms.tasks.zzn.run(Unknown Source:4)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7091)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
```

**Comment by kshivang** [comment]
This error may or may not occur, depend on build(may be related to jitpack). One workaround could be importing Goonj module in project directly

**Comment by kshivang** [comment]
Fixed in v0.5.8
